proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -46,6 +47,23 @@ message Repository {
   bool deprecated = 8;
   // deprecation_message is the message shown if the repository is deprecated.
   string deprecation_message = 9;
+  // owner_name is the name of the owner of the repository,
+  // either a username or organization name.
+  string owner_name = 10;
+  // description is the user configurable description of the repository.
+  string description = 11;
+  // url is the user configurable URL in the description of the repository,
+  // always included the scheme and will not have a #fragment suffix.
+  string url = 12;
+  // license is the user configurable license of the repository, which should be one of
+  // the identifier defined in https://spdx.org/licenses
+  string license = 13;
+  // license url is the user configurable url to the license text.
+  string license_url = 14;
+  // verification status of the owner of the repository, configurable by server admin.
+  VerificationStatus verification_status = 15;
+  // last commit time in main branch of the repository.
```

Comment:
nit: branches arent a bsr concept anymore, if we include this should it be something like "last main commit"?